package.json

```diff
@@ -22,7 +22,7 @@
     "@edx/frontend-auth": "5.3.6",
     "@edx/frontend-component-footer": "4.1.5",
     "@edx/frontend-logging": "2.1.0",
-    "@edx/paragon": "4.2.4",
+    "@edx/paragon": "^7.1.5",
```

Comment:
Should we still be specifying edx-bootstrap manually like we do?

Reply:
(Also, the lack of `^` carots is because we use renovate on the repo, and it hardcodes all the versions. So might want to keep that going.)

Reply:
No we should remove, I'll update and pin the paragon version